packages/dds/matrix/src/test/undoRedoStackManager.ts

```diff
@@ -48,6 +48,15 @@ class Stack<T> {
 			this.itemPushedCallback();
 		}
 	}
+
+	public get length(): number {
```

Comment:
Nit: same feedback as the other PR: docs please 😁 (for whichever PR goes in first)